### ISSUE TITLE
[A11y] Screenreader reads unrelated text before toast notification/alert contents

### ISSUE THREAD

**Issue by Benjin** [microsoft/vscode#81787]
Environment Details:
Application Name: Visual Studio Code
Application Version: 1.38.1 (user setup)
Commit: b37e54
Electron: 4.2.10
Node.js: 10.11.0
OS: Windows_NT x64 10.0.18362

Repro steps:
1. Go to Extensions
2. Pick any ol' extension (confirmed on Java Extension Pack)
3. Launch screen reader, like [NVDA](https://www.nvaccess.org/download/)
3. Use keyboard to select "Install" button
4. Hit <Enter> on keyboard

Expectation:
Toast content are read by the screen reader, as they're the most pertinent to the user's next action.

Reality:
Lots of intermediate text is read first, finally getting to the toast that's been waiting.  Should have higher aria priority.

**Comment by Benjin** [comment]
Refiling of https://github.com/microsoft/vscode/issues/81484 with repro specific to VS Code

**Comment by isidorn** [comment]
Assignign to next milestone to investigate then.

**Comment by isidorn** [comment]
Same reasoning as https://github.com/microsoft/vscode/issues/81786

I just tried this out and I disagree.
Notificatinos are something that is in the background and they should definentely not steal users attention by being read immediatly.

If every notification was doing this, then the user can be distributed all the time by random spam notifications from extensions. And this would lead to a very bad exeperience.

I am closing this as caused by extension and my recommendation is that the Java Extension Pack should use a modal dialog, and not a notificatino if the extension wants to focus to be switched to that and the user to be interupted.
I hope this makes sense, but all the API is there and it is up to the extension to decide which dialog should steal focus (use modal) and which not (use notifications).

Bottom line: notifications should not be obtrusive.
